tests/test_server_ops.py

```diff
@@ -178,6 +179,7 @@ async def test(host):
                 max_allowed_connections=None,
                 postgres_dsn=f'postgres:///?user=postgres&port={port}&'
                              f'host={host}',
+                runstate_dir=None if devmode.is_in_dev_mode() else host,
```

Comment:
Why are we passing `host` as `runstate_dir` here?

Reply:
Oh that's a bad naming - it's actually the path to a temp dir. Let me rename it.

Reply:
Oh that's a bad naming - it's actually the path to a temp dir. Let me rename it.